src/apps/Console/Console.js

```diff
@@ -188,7 +146,7 @@ function Console({ apps, wrapper }) {
         setLoading(false)
       }
     },
-    [toast, apps, wrapper, isConnected, web3, performIntents]
+    [toast, apps, wrapper, performIntents]
   )
   // Handle DAO execution
   const handleDaoExec = useCallback(
```

Comment:
I think it'd be really easy for us to migrate at least these handlers out to their own files to avoid having too much logic in this component. Everything in the `try` block can basically be reduced to an `async` function that we expect to return us a list of transactionPaths.

I think all these handlers need to do that (that can be passed in through the parameter), is just the `wrapper` and current `params`.

Doing this would then also allow us to greatly simplify `handleSubmit()` by using a mapping of command name to its handler function :) (and as noted earlier, we could take this even further by moving up all internal parsing and feedback UI rendering into those separate files; really similar to how the [individual `templates`](https://github.com/aragon/aragon/tree/master/src/templates) are structured).

Reply:
- [ ] `performIntents` function migrated
- [ ] handlers extracted